guides/common/attributes.adoc

```diff
@@ -1,13 +1,13 @@
 // Document state: "nightly" for master, "stable" for last two releases,
 // "unsupported" for the rest and "satellite" for satellite build
-:DocState: nightly
+:DocState: stable
 
 // Version numbers
-:ProjectVersion: nightly
+:ProjectVersion: 3.1
 :ProjectVersionPrevious: 3.0
-:KatelloVersion: nightly
-:TargetVersion: {ProjectVersion}
-:TargetVersionMaintainUpgrade: {ProjectVersion}
+:KatelloVersion: 4.3
+:TargetVersion: 7.0
+:TargetVersionMaintainUpgrade: 7.0
```

Comment:
You have Satellite version numbers but they should really be unchanged.

Here is where you'd change it for Satellite:

https://github.com/theforeman/foreman-documentation/blob/95433c67e36b1a616a9c8969567c83c5060361cf/guides/common/attributes-satellite.adoc#L14
https://github.com/theforeman/foreman-documentation/blob/95433c67e36b1a616a9c8969567c83c5060361cf/guides/common/attributes-satellite.adoc#L60-L61
https://github.com/theforeman/foreman-documentation/blob/95433c67e36b1a616a9c8969567c83c5060361cf/guides/common/attributes-satellite.adoc#L85-L86

Yes, it looks bad to have those all over the place rather than grouped together. I also don't know why there's so many variants for the versions.

Reply:
I copied those from the 3.0 attributes :see_no_evil:

Reply:
Perhaps that's just not updated since attributes were split.

Reply:
I think https://github.com/theforeman/foreman-documentation/pull/935 should improve that. It also missed the PreviousVersion update in master.